Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBProject.cs

```diff
@@ -19,7 +20,21 @@ public VBProject(VB.VBProject target, bool rewrapping = false)
 		
         public IApplication Parent => new Application(null);
 
-        public string ProjectId => HelpFile;
+        public string ProjectId
+        {
+            get
+            {
+                try
+                {
+                    return IsWrappingNullReference ? string.Empty : Target.ReadProperty("Rubberduck", "ProjectId");
+                }
+                catch (COMException)
+                {
```

Comment:
Note, there doesn't appear to be a clean way to check for property existence, hence catching ComEx.